gramps/gen/lib/serialize.py

```diff
@@ -71,3 +73,91 @@ def from_json(data):
     :rtype: object
     """
     return json.loads(data, object_hook=__object_hook)
+
+
+def to_struct(obj):
```

Comment:
Is it clearer to call this to_dict given it returns a Python dictionary?

Reply:
Yes, and I did call it that at one point. But I stole some code from the merge/diff code where I had called these "structs" for some reason. A long time ago :) I'll change it throughout.

Reply:
I do now :-)